modules/tls-profiles-kubernetes-configuring.adoc

```diff
@@ -0,0 +1,122 @@
+
+.Sample `APIServer` CR that configures the `Old` TLS security profile
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+ ...
+spec:
+  tlsSecurityProfile:
+    old: {}
+    type: Old
+ ...
+----
+
+The TLS security profile defines the the minimum TLS version and the TLS ciphers required by the Kubernetes control plane components to communicate with each other.
+
+You can see the ciphers and the minimum TLS version of the configured TLS security profile  in the `APIServer` custom resource (CR) under `Spec.Tls Security Profile` and `Status.Tls Profile`. 
+
+[IMPORTANT]
+====
+The Kubernetes control plane does not support TLS `1.3` and because the `Modern` profile requires TLS `1.3` it is not supported. The Kubernetes control plane converts the `Modern` profile to `Intermediate`.
+
+The Kubernetes control plane also converts the TLS `1.0` of an `Old` or `Custom` profile to `1.1`, and TLS `1.3` of a `Custom` profile to `1.2`.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Edit the default `APIServer` CR to configure the TLS security profile:
++
+[source,terminal]
+----
+$ oc edit APIServer cluster
+----
++
+.Sample `tls-security-profile.yaml` file
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: APIServer
+metadata:
+  name: cluster
+spec:
+  tlsSecurityProfile:
+    type: Custom <1>
+    custom: <2>
+      ciphers: <3>
+      - ECDHE-ECDSA-CHACHA20-POLY1305
+      - ECDHE-RSA-CHACHA20-POLY1305
+      - ECDHE-RSA-AES128-GCM-SHA256
+      - ECDHE-ECDSA-AES128-GCM-SHA256
+      minTLSVersion: VersionTLS11
+----
+<1> Specify the TLS security profile type. The default is `Intermediate`.
+* `Old`
+* `Intermediate`
+// * `Modern`
+* `Custom`
+<2> Specify:
+* `Old: {}`
+* `Intermediate: {}`
+// * `Modern: {}`
+* `Custom:`
+<3> For the `custom` type, specify a list of TLS ciphers and minimum accepted TLS version.
+
+.Verification
+
+* Verify that the TLS security profile is set in the `APIServer` CR:
++
+[source,terminal]
+----
+$ oc describe apiserver cluster
+----
++
+.Sample `APIServer` CR
+[source,yaml]
+----
+Name:         cluster
+Namespace:
+Labels:       <none>
+Annotations:  include.release.openshift.io/ibm-cloud-managed: true
+              include.release.openshift.io/self-managed-high-availability: true
+              include.release.openshift.io/single-node-developer: true
+              oauth-apiserver.openshift.io/secure-token-storage: true
+              release.openshift.io/create-only: true
+API Version:  config.openshift.io/v1
+Kind:         APIServer
+ ...
+Spec:
+  Audit:
+    Profile:  Default
+  Tls Security Profile:
+    Custom:
+      Ciphers:
+        ECDHE-ECDSA-CHACHA20-POLY1305
+        ECDHE-RSA-CHACHA20-POLY1305
+        ECDHE-RSA-AES128-GCM-SHA256
+        ECDHE-ECDSA-AES128-GCM-SHA256
+      Min TLS Version:  VersionTLS11
+    Type:               Custom
+ ...
+Status:
```

Comment:
This is a pretty long YAML snippet. Is it important to show the details both in the spec and status sections, or could you pick just one to show, and truncate the other?

Same for the ingress controller verification step